department-of-veterans-affairs/caseflow
# Refactors hearing conversion components so they can be shared

### Description
Related to #14894, refactors the hearing conversion components that will be used in the schedule veteran components so they can be shared between both

### Acceptance Criteria

### Testing Plan
1.  Run storybook locally `make storybook`
1. Ensure the RepresentativeSection, AppellantSection and HearingConversion components all still work as expected
